knowledge-base/number-in-the-cell-is-formatted-as-text-or-preceded-by-an-apostrophe.md

```diff
@@ -0,0 +1,56 @@
+---
+title: How to Avoid Warning in Excel The number in the cell is formatted as text or preceded by an apostrophe
```

Comment:
You may  add something to separate the error message, for example, a dash or brackets (I think they are not prohibited): How to Avoid Warning in Excel - The number in the cell is formatted as text or preceded by an apostrophe